lib/cc/analyzer/container.rb

```diff
@@ -124,6 +124,15 @@ def container_data(duration: nil, status: nil)
           @stderr_io.string
         )
       end
+
+      def reap_running_container
+        container_id = `docker ps --quiet --filter name=#{@name}`.chomp
```

Comment:
You're right! That's much better.

I had been getting errors in the specs which I had to fix by letting the container start before trying to reap it (hence the `sleep 1`, etc) and mistook that as meaning I had to use the ID -- it's been a confusing evening on this track....